server/src/main/java/org/opensearch/extensions/ExtensionsManager.java

```diff
@@ -308,7 +307,7 @@ private void discover() throws IOException {
                 logger.info("Loaded all extensions");
             }
         } else {
-            logger.info("Extensions.yml file is not present.  No extensions will be loaded.");
+            logger.error("Extensions.yml file is not present.  No extensions will be loaded.");
```

Comment:
Nit: make this WARN level.  This is a recoverable/handled condition.  See https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels

For users who don't use extensions, we don't want to scare them with a sudden ERROR in their logs that make them think something is broken.

Reply:
I like that, thank you.